### ISSUE TITLE
Deploying project doesn't update sidebar list

### ISSUE THREAD

**Issue by magicznyleszek** [kobotoolbox/kpi#1882]
This is a minor thing, as reloading page or clicking `Projects` to load list agains fixes this. Still would be nice to have this change live.

<img width="209" alt="screen shot 2018-07-24 at 11 30 03" src="https://user-images.githubusercontent.com/2521888/43129717-52a07368-8f35-11e8-988e-4683f1cfc5f9.png">

**Comment by pmusaraj** [comment]
Yes, this would be nice to have. But a more general thing we need to work on is to have operations on assets (delete, update, etc.) update the asset list store. Right now, we need to refresh the whole list after any changes, and it's inefficient. (We've long been wondering whethermoving to Redux would help with this, but haven't had the time to do that.)
